mithril-ui/menu/menu.css

```diff
@@ -0,0 +1,15 @@
+.Menu-container {
+  display: flex;
+  position: absolute;
+  z-index: 999;
```

Comment:
I would be careful with random numbers like this. Can we make it only as high as we need it to be?